docs/Spatial.md

```diff
@@ -1,6 +1,99 @@
 #Phidget Spatial Sensor
-More commonly known as an accelerometer.
+More commonly known as an accelerometer, the PhidgetSpatial library makes for intuitive and lightning fast development without any compromise. All spacial directions must be defined by user, for an in dept guide into this, [here is a great guide](http://www.phidgets.com/docs/1044_User_Guide). For a quick start into your Spatial project see this [basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js).
 
-[Basic spatial example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/spatial.js)
+##Methods
 
-ToDo : map out module and describe use
\ No newline at end of file
+The below information lists the parameters, events, and give you further information about your __Phidgets Spatial__. For low level development, [see here](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams).
+
+|Method call|Parameters|Description|
+|---|---|---|
+|[__phidget__.connect](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|[__phidget__.params object](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|This connects the Phidget Spatial|
+|[__phidget__.whenReady](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|function |This executes a function when the Phidgets RFID is ready to be used.|
+|[__phidget__.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
+|__phidget__.observeGyro|'gyro'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the gyroscope on the __Spatial__ board. |
+|__phidget__.observeAcceleration|'accel'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the acceleration sensors on the __Spatial__ board. |
+|__phidget__.observeMagneticField|'mag'|This [observes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe) for the magnetic sensors, needed for the compass feature, on the __Spatial__ board. |
+
+##Data
+ 
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|This is an identifier, Phidget GPS will always return 'PhidgetSpatial'|
+|accelerationMin|number|no|This returns the maximum negative acceleration that the desired axis will report.|
```

Comment:
Calling this negative acceleration is confusing and over complicating. If it returns a negative number it is clear that it is reverse acceleration. No need to clarify that a negative number is negative.